docs/runbooks/monthly-release-to-editors-and-webmasters.md

```diff
@@ -20,36 +20,56 @@ release - 1. This usually mean last weeks release.
-    1. The 'x-defaults' anchors' 'dpl-cms-release' tag should be bumped to the
-    new version.
-    2. Then update the 'moduletest-dpl-cms-release' of 'x-webmaster'
-    and 'x-webmaster-with-defaults' anchor.
-    3. Now update 'dpl-cms-release' of the 'x-webmaster' anchor to the penultimate
-    version of the cms.
-4. Commit the change and push your branch to GitHub and create a pull request.
-5. Request a review for the change and wait for approval.
-6. Start `dplsh` from the `/infrastructure` directory of your local
-   environment by running `../tools/dplsh/dplsh.sh`
-7. Run `task cluster:mode:release` to get more nodes while deploying.
-8. Run `task sites:sync` from `dplsh` to deploy the changes.
-9. If there are any Terraform changes then do not apply them, abort the
-   deployment and consult the platform team.
-10. Open the Deployments list page within the Lagoon UI to see all running and
-   queued deployments.
-11. Wait for all the deployments to complete.
-12. Run `task sites:incomplete-deployments` to identify any failed deployments.
-13. If some deployments did not complete determine if the error relates to the
-    platform or the application.
-14. For all platform-related errors try to redeploy the environment from
-    the Lagoon UI.
-15. Merge the pull request once the deployment completes.
-16. Run `task cluster:adjust:resource-request` from `dplsh`.
-17. Run `task cluster:promote-workloads-to-prod` from `dplsh`.
-18. Reset the webmaster moduletest environments.
-19. Run 'task cluster:mode:reset' from `dplsh`.
-
-## Procedure: a some sites fails to deploy
+3. Release to webmaster production:
+    1. Update 'dpl-cms-release' of the 'x-webmaster' anchor to the penultimate
+        version of the CMS.
+    2. Commit the change and push your branch to GitHub and create a pull request.
+    3. Request a review for the change and wait for approval.
+    4. Start `dplsh` from the `/infrastructure` directory of your local
+       environment by running `../tools/dplsh/dplsh.sh`
+    5. Run `task cluster:mode:release` to get more nodes while deploying.
+    6. Run `task sites:sync` from `dplsh` to deploy the changes.
+    7. If there are any Terraform changes then do not apply them, abort the
+       deployment and consult the platform team.
+    8. Open the Deployments list page within the Lagoon UI to see all running and
+       queued deployments.
+    9. Wait for all the deployments to complete.
+    10. Run `task sites:incomplete-deployments` to identify any failed deployments.
+    11. If some deployments did not complete determine if the error relates to the
+        platform or the application.
+    12. For all platform-related errors try to redeploy the environment from
+        the Lagoon UI.
+    13. Merge the pull request once the deployment completes.
+    14. Synchronize moduletest with main,so moduletest becomes an exact
+        copy of main by running `task sites:webmaster:reset-moduletest`.
```

Comment:
As I see it this is a new step :+1:

Reply:
it is, setting the cluster into release mode and back is also relatively new